tests/integration_tests/functional/test_cpu_features.py

```diff
@@ -186,25 +187,47 @@ def test_brand_string(test_microvm_with_api, network_config):
     "0xc0000083",  # MSR_CSTAR
     "0xc0000100",  # MSR_FS_BASE
     "0xc0000101",  # MSR_GS_BASE
+    # MSRs below are required on T2A but we
+    # we are adding them to the common exception list to keep things simple
+    "0x834"     ,  # LVT Performance Monitor Interrupt Register.
+    "0xc0010007",  # MSR_K7_PERFCTR3
+    "0xc001020b",  # Performance Event Counter MSR_F15H_PERF_CTR5
+    "0xc0011029",  # MSR_F10H_DECFG also referred to as MSR_AMD64_DE_CFG
 ]
+
+# CPU templates which support MSR test
+MSR_SUPPORTED_TEMPL = ["T2S", "T2A"]
+MSR_SUPPORTED_VENDORS = [
```

Comment:
nit: if you prefer to leave it, I don't think we need to disable formatting check, shouldn't be too ugly.